### PR TITLE
[Documentation] Image update for alcor_build

### PR DESCRIPTION
**Pull request by yanmo96**
Added the new alcor_build.png, 
Changed readme to the new png.
Removed the alcor_build.jpg.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ $  mvn clean install
 
 You could expect to see _BUILD SUCCESS_ at the end of the build console.
 
-![Alcor Build](modules/ROOT/images/alcor_build.JPG)
+![Alcor Build](modules/ROOT/images/alcor_build.png)
```
Comment (xieus) [inline docs/README.md:67]: This image looks a bit large. Can you adjust the size a bit? 

You can check the effect here:
https://github.com/yanmo96/alcor/blob/image_update/docs/README.md

**Review by xieus — COMMENTED** [review]
@yanmo96 Thanks for making the change. A minor comment, please take a look.

File: docs/README.md (hunk 1 above)
Comment (yanmo96) [reply docs/README.md]: Just updated the readme file to limit to 720px. Hope it looks ok now :)

**Review by xieus — APPROVED** [review]
LGTM.

**Pull request merged**